src/main/java/no/nav/pto/veilarbportefolje/oppfolging/OppfolgingStartetService.java

```diff
@@ -3,61 +3,26 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import no.nav.common.types.identer.AktorId;
-import no.nav.pto.veilarbportefolje.database.BrukerRepository;
-import no.nav.pto.veilarbportefolje.domene.AktorClient;
-import no.nav.pto.veilarbportefolje.domene.value.PersonId;
-import no.nav.pto.veilarbportefolje.kafka.KafkaCommonConsumerService;
 import no.nav.pto.veilarbportefolje.opensearch.OpensearchIndexer;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.time.ZonedDateTime;
-import java.util.List;
-import java.util.NoSuchElementException;
 
 
 @Slf4j
 @Service
 @Transactional
 @RequiredArgsConstructor
-public class OppfolgingStartetService extends KafkaCommonConsumerService<OppfolgingStartetDTO> {
+public class OppfolgingStartetService {
     private final OppfolgingRepository oppfolgingRepository;
     private final OppfolgingRepositoryV2 oppfolgingRepositoryV2;
     private final OpensearchIndexer opensearchIndexer;
-    private final BrukerRepository brukerRepository;
-    private final AktorClient aktorClient;
-
-    @Override
-    public void behandleKafkaMeldingLogikk(OppfolgingStartetDTO dto) {
-        mapAktoerTilPersonId(dto.getAktorId());
-        startOppfolging(dto.getAktorId(), dto.getOppfolgingStartet());
-    }
 
     public void startOppfolging(AktorId aktorId, ZonedDateTime oppfolgingStartetDate) {
         oppfolgingRepository.settUnderOppfolging(aktorId, oppfolgingStartetDate);
         oppfolgingRepositoryV2.settUnderOppfolging(aktorId, oppfolgingStartetDate);
         opensearchIndexer.indekser(aktorId);
         log.info("Bruker {} har startet oppfølging: {}", aktorId, oppfolgingStartetDate);
     }
-
-    private void mapAktoerTilPersonId(AktorId aktorId) {
```

Comment:
This code should not be removed. this is the code that makes data from arena mappable to aktoerId

Reply:
Maybe move it to the new service?

Reply:
Or to startOppfolging method

Reply:
so this mapping should be on oppfolging start..

Reply:
Yes, but it does not matter if we also map on oppfølging avsluttet. 
The plan is to do this mapping based on a topic from PDL, but that work is a bit far away